google_shopping_functions.js

```diff
@@ -2,9 +2,63 @@
 //accepts full item data
 //returns the length of the items array
 var getItemsCount = function(itemData){
-	return itemData.items.length;
+
+	return itemData.length;
 };
 
```

Comment:
This first function looks good, it just needs a bit of reformatting. The more standard practice would be to write it like so:

```
var getItemsByBrand = function(items, brand) {
    var array = [];
    for (var l = 0; l < items.length; l++) {
        var brnd = items[l]["product"].brand;
        if (brnd.toLowerCase() === brand.toLowerCase()) {
            array.push(items[l]);
        };
    };
}